interface/src/app/tribe_client/geneset-search-bar.tpl.html

```diff
@@ -1,12 +1,11 @@
 <div>
   <form class="form-inline">
     <label>Organism:</label>
-    <select ng-disabled="true">
-      <option ng-selected="true" value="organism">{{organism}}</option>  
-    </select>
+    <i>{{$ctrl.organism}}</i>&nbsp;
+    <input ng-model="$ctrl.search.query" placeholder="Search text">
```

Comment:
Continuing my sleep-deprived simulation of a dumb user: what index are we actually searching here? I tried some gene names but using genes that came up as hits under other more general searches (e.g. "dna") do not return results. Some things I tried: "mexR", "PA2196".